### ISSUE TITLE
I can not get your localization result in chess of 7Scenes when I train

### ISSUE THREAD

**Issue by zhuixunforever** [BingCS/AtLoc#3]
thank you for your nice work
I want to train chess of 7Scenes by myself， and use the fellow parameters:

>python train.py --dataset 7Scenes --scene chess --model AtLoc --gpus 0 --epochs 1000 --color_jitter 0.0

save models every 5 epochs, and I get 200 models, I test these model by run eval.py(I made a little modify to loop to load these 200 models)

> python eval.py --dataset 7Scenes --scene chess --model AtLoc --gpus 0 --weights ./logs/7Scenes_chess_AtLoc_False/models/

and, what I get min translation median or min rotation median is:
epoch|tran_median|tran_mean|rota_median|rota_mean
epoch_570 | 0.11 | 0.16 | 4.97 | 6.25
epoch_620 | 0.12 | 0.16 | 4.88 | 6.21

but your result in paper of AtLoc is:
tran_median|rota_median
0.10|4.07

I don't know why, can you tell me how did you train?

**Comment by BingCS** [comment]
Hi @zhuixunforever, thanks for your interets in our work. You are encouraged to train several times. The learning rate can be set as 0.0001 to easily get equal or better results on 7Scenes.

**Comment by zhuixunforever** [comment]
@BingCS thank you for your reply, and I'll try to use learning rate with 0.0001 and set epoch 1000.

**Comment by dyz-zju** [comment]
Hi，I also want to train the model on 7 Scenes by myself, but I can't get the same results as your paper. Could you tell me how to use your code to get the results?
I train stairs of 7 Scenes, and use the fellow parameters:
python train.py --dataset 7Scenes --scene stairs --model AtLoc --gpus 0 --epochs 1000 --color_jitter 0.0 --lr 0.0001
my train results: 0.24 12.90
your paper results: 0.26 10.5
@BingCS @zhuixunforever

**Comment by BingCS** [comment]
Hi @dyz-zju, the trained model of Stairs can be accessed via https://drive.google.com/drive/folders/1inY29zupeCmvIF5SsJhQDEzo_jzY0j6Q. Please also note that, the minimum val_loss (MSE Loss) doesn't correspond to the best performance. This is because the metric used for reporting is by convention the median errors[1,2]. 

[1] Kendall, Alex, Matthew Grimes, and Roberto Cipolla. "Posenet: A convolutional network for real-time 6-dof camera relocalization." CVPR 2015.
[2] Brahmbhatt, Samarth, et al. "Geometry-aware learning of maps for camera localization." CVPR 2018.

**Comment by dyz-zju** [comment]
Thanks for your reply, I want to train it by myself, could you give me some advices?

**Comment by dyz-zju** [comment]
> thank you for your nice work
> I want to train chess of 7Scenes by myself， and use the fellow parameters:
> 
> > python train.py --dataset 7Scenes --scene chess --model AtLoc --gpus 0 --epochs 1000 --color_jitter 0.0
> 
> save models every 5 epochs, and I get 200 models, I test these model by run eval.py(I made a little modify to loop to load these 200 models)
> 
> > python eval.py --dataset 7Scenes --scene chess --model AtLoc --gpus 0 --weights ./logs/7Scenes_chess_AtLoc_False/models/
> 
> and, what I get min translation median or min rotation median is:
> epoch|tran_median|tran_mean|rota_median|rota_mean
> epoch_570 | 0.11 | 0.16 | 4.97 | 6.25
> epoch_620 | 0.12 | 0.16 | 4.88 | 6.21
> 
> but your result in paper of AtLoc is:
> tran_median|rota_median
> 0.10|4.07
> 
> I don't know why, can you tell me how did you train?

Have you got the similar results as papers?
